src/preferences/dialog/dlgprefsounddlg.ui

```diff
@@ -231,9 +231,6 @@
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
-         <property name="toolTip">
-          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Warning!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Using multi threading may result in pitch and tone imperfaction depending of the platform, leading to mono-incompatibiltiy, due to third party limitations. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
The tooltip was not to bad. Since we have now the message box we can sharpen it a bit: 
"Using dual-threaded stereo may improve CPU utilization, but results to mono-incompatibility (Therefore not recommended)."
Without any formatting.

Reply:
The tooltip is injected dynamically in the UI, this was already the case, and this hardcoded tooltip was redundant. See `DlgPrefSound`